src/sites/cclw/constants/faqs.tsx

```diff
@@ -0,0 +1,353 @@
+import { ExternalLink } from "@components/ExternalLink";
+import { LinkWithQuery } from "@components/LinkWithQuery";
+
+type TFAQ = {
+  title: string;
+  content: JSX.Element;
+};
+
+export const FAQS: TFAQ[] = [
```

Comment:
Suggestion - we could put the FAQs into a json file or similar - just to separate content from formatting?